miner/bid_simulator.go

```diff
@@ -78,6 +78,7 @@ type simBidReq struct {
 type bidSimulator struct {
 	config        *MevConfig
 	delayLeftOver time.Duration
+	minGasPrice   *big.Int
```

Comment:
no need for this config. we reuse the config in the miner module.

Reply:
it is